app/src/main/java/ch/epfl/sdp/musiconnect/pages/ProfileModificationPage.java

```diff
@@ -231,7 +237,7 @@ private void btnSave(String[] newFields) {
 
         // Upload video to cloud storage
         if (videoRecorded) {
```

Comment:
Store the string value in the String.xml file

Reply:
sure